architecture/kubernetes_infrastructure.adoc

```diff
@@ -9,30 +9,43 @@
 
 toc::[]
 
-== Master
+Kubernetes is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenence, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications. 
 
-The host(s) containing the master components (API Server, Controller Manger Server, etcd, etc)
+A Kubernetes cluster involves a master component and a set of nodes, all on top of a distributed storage system. 
```

Comment:
Saying a master component makes it sound like there is 1 instance.  And there isn't any requirement of a distributed storage system it all runs on top of.  Containers may use storage.  And that storage should be some sort of network storage.  But it doesn't need to be the backbone of the masters or the nodes themselves.